src/pages/docs/octopus-cloud/frequently-asked-questions.md

```diff
@@ -0,0 +1,191 @@
+
+### Where can I learn more about pricing?
+Please refer to our dedicated [Octopus pricing](https://octopus.com/pricing/overview) overview and our [Octopus Cloud pricing FAQ](https://octopus.com/pricing/faq#octopus-cloud-section). Alternatively, please [contact our Sales team](https://octopus.com/company/contact) who can answer any other questions you may have.
+
+## Data Centers
+
+### Where is Octopus Cloud hosted? 
+We [host Octopus Cloud](/docs/octopus-cloud#octopus-cloud-hosting-locations) in the following Azure regions:
+* West US 2
+* West Europe
+* Australia East
+  
+Each customer's Octopus Cloud is wholly located within a single Azure region.
+
+### Where are the data centers located? 
+Each customer's Octopus Cloud is wholly located within a single Azure region. Microsoft publishes [data center information](https://datacenters.microsoft.com/) for each Azure region. 
+
+### Can we choose another Azure region?
+Please [contact our sales team](https://octopus.com/company/contact) to discuss this.
+
+### Can I move my Octopus Cloud to another region?
+Yes. [Contact Support]((https://octopus.com/company/contact)) to arrange the relocation.
+
+## Security
+
+### How does Octopus Cloud approach security?
+We pride ourselves on making Octopus Deploy a [secure product](/docs/security#octopus-cloud). In addition, Octopus Cloud instances are [secure and compliant](/docs/octopus-cloud#secure-and-compliant-out-of-the-box) out of the box. 
+
+### Is Octopus Cloud compliant with GDPR?
+Octopus complies with the European Union's General Data Protection Regulation ([GDPR](https://octopus.com/legal/gdpr)). 
+
+### Is Octopus Cloud compliant with SOC 2?
+We are regularly audited and maintain a SOC 2 Type II certification. You can request access to our confirmation from our [Trust Center](https://octopus.com/company/trust).
+
+### Is Octopus Cloud compliant with ISO 27001?
+Our information security management system has been assessed and found to conform with ISO 27001:2013. You can view our certificate via our [Trust Center](https://octopus.com/company/trust).
+
+### Is Octopus Cloud compliant with HIPAA?
+Octopus Deploy is not a covered entity under HIPAA. We do not process personally identifiable health data directly or on behalf of our customers. Octopus Cloud has global healthcare industry customers operating on our platform. 
+
+### Where can I learn more about Octopus Deploy compliance?
+The security and privacy of our customers' personal, company, and intellectual property data are top priorities at Octopus. We're dedicated to maintaining and continuously improving our security and compliance programs, which we publish on our [Trust Center](https://octopus.com/company/trust).
+
+### Which authentication providers are supported?
+Octopus Cloud supports the most common [authentication providers](/docs/security/authentication/auth-provider-compatibility#login-support) out of the box.
```

Comment:
It only makes sense to describe 'Octopus Deploy' (i.e. the overall product encapsulating both Octopus Server and Octopus Cloud) as supporting the most common auth providers out of the box, i.e. like this doc: https://octopus.com/docs/security/authentication.

Octopus Cloud doesn't support Active Directory or LDAP Authentication as shown in the table this section links to.

Reply:
Will tighten the answer.